sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -265,6 +265,7 @@ predicate
     | predicate IS nullNotnull                                      #isNullPredicate
     | left=predicate NOT? LIKE right=predicate                      #likePredicate
     | left=predicate REGEXP right=predicate                         #regexpPredicate
+    | predicate NOT? IN predicateList                               #inList
```

Comment:
np: just found logical expression is also valid. Reference: https://github.com/antlr/grammars-v4/blob/master/sql/mysql/Positive-Technologies/MySqlParser.g4#L2454

Reply:
Done.